opencensus/trace/propagation/b3.h

```diff
@@ -0,0 +1,63 @@
+// Copyright 2019, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef OPENCENSUS_TRACE_PROPAGATION_B3_H_
+#define OPENCENSUS_TRACE_PROPAGATION_B3_H_
+
+#include <string>
+
+#include "absl/strings/string_view.h"
+#include "opencensus/trace/span_context.h"
+
+namespace opencensus {
+namespace trace {
+namespace propagation {
+
+// Implementation of the B3 propagation format:
+// https://github.com/openzipkin/b3-propagation
+
+// We handle:
```

Comment:
We should probably handle https://github.com/openzipkin/b3-propagation#debug-flag as well.

Reply:
Done.